### ISSUE TITLE
RF & GBDT Add more Loss Function

### ISSUE THREAD

**Issue by zhangpengshan** [ShifuML/shifu#219]
Like HuberLoss and GuantileLoss in scikit-learn
